addons/website_event_track/data/mail_template_data.xml

```diff
@@ -29,4 +29,163 @@
         <field name="lang">{{ object.partner_id.lang }}</field>
         <field name="auto_delete" eval="True"/>
     </record>
-</data></odoo>
+    
+    <record id="email_reminder" model="mail.template">
+        <field name="name">Add reminder via email</field>
+        <field name="model_id" ref="website_event_track.model_event_track"/>
+        <field name="subject">Add talk reminder: {{ object.name }}</field>
+        <field name="email_from">{{ (object.event_id.organizer_id.email_formatted or object.event_id.company_id.email_formatted or '') }}</field>
+        <field name="description">Add talk reminder</field>
+        <field name="body_html" type="html">
+<table border="0" cellpadding="0" cellspacing="0" style="padding-top: 16px; background-color: #F1F1F1; font-family:Verdana, Arial,sans-serif; color: #454748; width: 100%; border-collapse:separate;"><tr><td align="center">
+<t t-set="date_begin" t-value="format_datetime(object.event_id.date_begin, tz='UTC', dt_format=&quot;yyyyMMdd'T'HHmmss'Z'&quot;)"/>
+<t t-set="date_end" t-value="format_datetime(object.event_id.date_end, tz='UTC', dt_format=&quot;yyyyMMdd'T'HHmmss'Z'&quot;)"/>
+<t t-set="is_online" t-value="'is_published' in object.event_id and object.event_id.is_published"/>
+<t t-set="is_sale" t-value="'sale_order_id' in object and object.sale_order_id"/>
+<t t-set="event_organizer" t-value="object.event_id.organizer_id"/>
+<t t-set="event_address" t-value="object.event_id.address_id"/>
+<t t-set="registration_ids" t-value="object.ids if not is_sale else object._get_event_registration_ids_from_order()"/>
+<table border="0" cellpadding="0" cellspacing="0"  width="590" style="padding: 16px; background-color: white; color: #454748; border-collapse:separate;">
```

Comment:
Note that there should be no styling on mail templates, instead you should use a layout notification xml id that makes the layouting, rendered at sending time. You can look at email_layout_xmlid field on mail.template for example :)